app/src/main/res/values-zh-rTW/about.xml

```diff
@@ -0,0 +1,628 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about_title">關於</string>
+    <string name="about_close">關閉</string>
+    <string name="about_whoami">I am monerujo</string>
+    <string name="about_version">版本 %1$s (%2$d)</string>
+
+    <string name="credits_text"><![CDATA[
+        <b>致謝</b>
+        <br/>
+        m2049r, baltsar777, anhdres, keejef,
+        rehrar, EarlOfEgo et al.
+        <br/><br/>
+        <a href="https://monerujo.io/">monerujo.io</a>
+        ]]></string>
+
+    <string name="privacy_policy"><![CDATA[
```

Comment:
is there a reason why the privacy policy is not translated?

Reply:
I worried about the translation quality and wording might bring some responsibility issue in policy and license content. I have little experience here so I left them in original text. Do you have suggestion?

Reply:
I found @0140454 have translated it, haha. If @m2049r doesn't think that's an issue I'll include [the PR](https://github.com/Lafudoci/xmrwallet/pull/1/files). 0140454 made a great translation work on that.

Reply:
licenses i wouldn't touch, but the privacy policy should be fine. please include it. thank you!!

Reply:
OK, I have reviewed @0140454 's work and merged.